modules/telco-troubleshooting-security-authentication.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/troubleshooting/telco-troubleshooting-security.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="telco-troubleshooting-security-authentication_{context}"]
+= Authentication
+
+Determine which identity provider or providers are in your cluster.
```

Comment:
Suggest "identity providers" instead of "identity provider or providers" to be less wordy.